src/auth/constants.ts

```diff
@@ -0,0 +1,7 @@
+import 'dotenv/config';
+
+export const BCRYPT = {
+  salt: +process.env.BCRYPT_SALT_ROUNDS!,
+};
+
+export const MAX_INT32 = 2147483647;
```

Comment:
could also be written as
```suggestion
export const MAX_INT32 = Math.pow(2, 31) - 1;
```